addons/binding/org.openhab.binding.hs110/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Since a few weeks every file should have a reference to the about.html, for an example check the current master.